_posts/2015-06-07-cosmic-neutrinos.md

```diff
@@ -0,0 +1,45 @@
+However, there may be alternative explanations as to why this is the case; these explanations can be tested by probing the composition, and therefore the sources, of the UHECRs.
+
+One such probe involves searching for ultrahigh-energy cosmic neutrinos.
+Neutrinos are particles belonging to the same family as the electron but are extremely weekly interacting, meaning that they can pass through lightyears of lead without interacting in any way with the dense metal.
+If these neutrinos are produced through the interactions of “primary particles” of cosmic rays with the CMB, they will provide valuable information about the nature of the primary particles, since protons are expected to produce greater numbers of neutrinos through these interactions than heavier primary particles.
+On the other hand, cosmic neutrinos that are the result of the transformation of charged [pions](https://simple.wikipedia.org/wiki/Pion) produced at the cosmic-ray source may be the only cosmic-ray particles capable of travelling distances greater than around 100 megaparsecs or 360 million lightyears.
+Observing and studying these kinds of neutrinos would advance our understanding of astrophysics and help validate or rule out some of the theories that attempt to explain the UHECR energy spectrum and help determine the nature of very distant cosmic-ray sources.
+
+The [Pierre Auger Observatory](https://www.auger.org/) in Argentina has a detection area of around 3000 km<sup>2</sup> and is operated by an international collaboration of more than 500 scientists.
+The Surface Detector (SD) array of the observatory can detect neutrinos that have energies of around and over 1 exaelectronvolt (10<sup>18</sup> eV), which is about [150,000 times the energy of the proton beams in the Large Hadron Collider](http://www.wolframalpha.com/input/?i=1+EeV+vs+6.5+TeV).
+The neutrinos are not observed directly but through cascading showers of particles (mostly electrons or their heavier cousins known as [muons](https://simple.wikipedia.org/wiki/Muon)) produced when the neutrinos interact with the Earth’s atmosphere.
+
+![Three of the stations of the Surface Detectors in the Pierre Auger Observatory]({{ site.url }}/images/2015-06-07-cosmic-neutrinos.jpg)
+
+> _Three of the stations of the Surface Detectors in the Pierre Auger Observatory in a row towards the Andes, at sunrise. [Image](https://commons.wikimedia.org/wiki/File:SD_tanks_and_Andes.JPG) by Lorenzo Caccianiga / [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/)_
+
+The Auger team analysed data collected by the SD array from 1 January 2004 until 20 June 2013, which, excluding the periods of so-called “data-acquisition instabilities”, corresponds to around 6.4 years of continuous operation.
+During this period, not a single candidate for neutrino-induced showers was observed by the array in the spatial orientation in which the search was carried out.
```

Comment:
During this period, they didn't observe...
Remove spatial orientation, irrelevant to lay reader